kubeflow/hack/setup_oauth_client.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# A helper script to create OAuth clients using the caller's GCP project.
+#
+
+if [[ ! -n $ADMIN_EMAIL ]]; then
+  echo "Please provide a value for ADMIN_EMAIL"
+  exit 1
+fi
+
+if [[ ! -n $NAME ]]; then
```

Comment:
We use `KF_NAME` as a name for the Kubeflow cluster. Can we rename this variable into `OAUTH_CLIENT_NAME`?

Reply:
The OAuth client name is not really significant and one seems sufficient for a domain, should we just use some default name here?